### PR TITLE
[postgres] Support metadata columns for postgres-cdc connector

### PR DESCRIPTION
**Pull request by GOODBOY008**
Expose following Postgres metadata columns in this PR:

issue: https://github.com/ververica/flink-cdc-connectors/issues/249

<html>
<body>

Key | DataType | Description
-- | -- | --
database_name | STRING NOT NULL | Name of the database that contain the row.
collection_name | STRING NOT NULL | Name of the collection that contain the row.
op_ts | TIMESTAMP_LTZ(3) NOT NULL | It indicates the time that the change was made in the database. If the record is read from snapshot of the table instead of the change stream, the value is always 0.

</body>
</html>

### CODE REVIEW

**Comment by GOODBOY008** [comment]
Hi,@wuchong @leonardBang 
 If you are available, please help code rebview. Sorry for blocking meta data issue.Becaulse of my incoming ACP exame.

File: flink-connector-postgres-cdc/src/test/java/com/ververica/cdc/connectors/postgres/table/PostgreSQLConnectorITCase.java [hunk 1]
```diff
@@ -44,7 +45,7 @@
 import static org.junit.Assert.assertTrue;
 import static org.testcontainers.containers.PostgreSQLContainer.POSTGRESQL_PORT;
 
-/** Integration tests for MySQL binlog SQL source. */
+/** Integration tests for PostgreSQL SQL source. */
```
Comment (leonardBang) [inline flink-connector-postgres-cdc/src/test/java/com/ververica/cdc/connectors/postgres/table/PostgreSQLConnectorITCase.java:48]:
```suggestion
/** Integration tests for PostgreSQL Table source. */
```

File: flink-connector-postgres-cdc/src/main/java/com/ververica/cdc/connectors/postgres/table/PostgreSQLReadableMetadata.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+import com.ververica.cdc.debezium.table.MetadataConverter;
+import io.debezium.connector.AbstractSourceInfo;
+import io.debezium.data.Envelope;
+import org.apache.kafka.connect.data.Struct;
+import org.apache.kafka.connect.source.SourceRecord;
+
+/** Defines the supported metadata columns for {@link PostgreSQLTableSource}. */
+public enum PostgreSQLReadableMetadata {
+    /** Name of the table that contain the row. . */
```
Comment (leonardBang) [inline flink-connector-postgres-cdc/src/main/java/com/ververica/cdc/connectors/postgres/table/PostgreSQLReadableMetadata.java:34]:
```suggestion
    /** Name of the table that contain the row. */
```

Comment (leonardBang) [reply flink-connector-postgres-cdc/src/main/java/com/ververica/cdc/connectors/postgres/table/PostgreSQLReadableMetadata.java:34]: Appreciate if you can help update the mysql connector. ^_^

File: flink-connector-postgres-cdc/src/test/java/com/ververica/cdc/connectors/postgres/table/PostgreSQLTableFactoryTest.java [hunk 3]
```diff
@@ -60,6 +61,18 @@
...
 
+    private static final ResolvedSchema SCHEMA_WITH_METADATA =
+            new ResolvedSchema(
+                    Arrays.asList(
+                            Column.physical("id", DataTypes.BIGINT().notNull()),
+                            Column.physical("name", DataTypes.STRING()),
+                            Column.physical("count", DataTypes.DECIMAL(38, 18)),
+                            Column.metadata("time", DataTypes.TIMESTAMP(3), "op_ts", true),
+                            Column.metadata(
+                                    "_database_name", DataTypes.STRING(), "database_name", true)),
```
Comment (leonardBang) [inline flink-connector-postgres-cdc/src/test/java/com/ververica/cdc/connectors/postgres/table/PostgreSQLTableFactoryTest.java:72]: minor:  we usually use "_" prefix for column name in Mongo DB

**Review by leonardBang — COMMENTED** [review]
Thanks @GOODBOY008 for the nice work, I only left several minor comments. And we should update the documentation when we introduce new feature. Appreciate if you can also update the documentation.

**Comment by GOODBOY008** [comment]
> Thanks @GOODBOY008 for the nice work, I only left several minor comments. And we should update the documentation when we introduce new feature. Appreciate if you can also update the documentation.

Thanks @leonardBang code review for my pr. I have fixed all problems.By the way,mysql connetor is update too.

**Review by leonardBang — APPROVED** [review]
Thanks @GOODBOY008 for the great work, it's cool that you also update the `mysql-cdc` documentation. LTGM once the CI green.

**Pull request merged**